recipes/lru-dict/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "lru-dict" %}
+{% set version = "1.1.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 365457660e3d05b76f1aba3e0f7fedbfcd6528e97c5115a351ddd0db488354cc
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
```

Comment:
You're going to need a `build` section that includes the C compiler like in:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L38-L41